nemo/collections/asr/parts/submodules/spectr_augment.py

```diff
@@ -44,15 +45,15 @@ def input_types(self):
         """Returns definitions of module input types
         """
         return {
-            "input_spec": NeuralType(('B', 'D', 'T'), SpectrogramType()),
```

Comment:
Could you revert these changes ?

Reply:
Done... this was `black`s default (string normalization) that was done automatically